en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **RTL Trigger based on remaining flight range ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and destination distance/direction
+* **Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))**
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
```

Comment:
MAVLink Ethernet has NO DOCS. Can we get some. Possibly needs to be added to http://docs.px4.io/master/en/peripherals/mavlink_peripherals.html - someone needs to write these or at least talk to me. 

For now, this needs release note needs list of the affected parameters.

Reply:
Can you raise an issue for this? It's out of my scope, I wouldn't know where to start even creating the issue